kubemarine/procedures/upgrade.py

```diff
@@ -137,6 +137,19 @@ def upgrade_containerd(cluster: KubernetesCluster):
                 if isinstance(value, dict):
                     config_string += f"\n[{key}]\n{toml.dumps(value)}"
             utils.dump_file(cluster, config_string, 'containerd-config.toml')
+
+            for node in cluster.nodes['control-plane'].include_group(
+                        cluster.nodes.get('worker')).get_ordered_members_list(
+                        provide_node_configs=True):
+                kubeadm_flags_file = "/var/lib/kubelet/kubeadm-flags.env"
+                pause_version = cluster.globals['compatibility_map']['software']['pause'][target_kubernetes_version]['version']
+                
+                kubeadm_flags = node['connection'].sudo(f"cat {kubeadm_flags_file}",
+                                        is_async=False).get_simple_out()
+                updated_kubeadm_flags = kubernetes._config_changer(kubeadm_flags, f"pause:{pause_version}",delemeter=":")
```

Comment:
```suggestion
                updated_kubeadm_flags = kubernetes._config_changer(kubeadm_flags, f"--pod-infra-container-image={sandbox}")
```
Another options using `=` as delimiter.